psi4/src/psi4/libmints/orthog.h

```diff
@@ -34,30 +34,56 @@
 
 namespace psi {
 
-class PSI_API OverlapOrthog {
+class PSI_API OverlapOrthogonalization {
```

Comment:
I was thinking that even though the class name is somewhat self-explanatory, it might be a good idea to add a comment that explains what this class does and how/where it is used. Just thinking about the future programmers.